module05/__init__.py

```diff
@@ -0,0 +1 @@
+import calculator
```

Comment:
You do not need **__init__.py** to complete the task. You can simply use the same import in the main file you create for the task. Your problem is very similar to what Marian had.

Reply:
Thank you for the comment.

I've used it to avoid error: `ModuleNotFoundError: No module named 'test_calculator.calculator'; 'test_calculator' is not a package` when running `test.py`. 

After investigation found that this error happens, because imported only functions in 
`test_calculator.py`, i.e.:
```
from calculator import add, subtract, multiply, divide
```
This way the functions could e used without `calculator.` in `test_calculator.py` the same way was used in the `calculator.py`.

You are right that if 
```
import from calculator
```
was used then  the error wouldn't happen and this file wouldn't be needed.